lib/DDG/Spice/Emojipedia.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Emojipedia;
+
+# ABSTRACT: Provide definitions and meanings for emoji characters.
+
+use strict;
+use utf8;
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'http://api.emojipedia.org/emojis/$1?api_key={{ENV{DDG_SPICE_EMOJIPEDIA_APIKEY}}}';
```

Comment:
> format=json shouldn't be required here; a default Accept header of / will return a JSON response. It's no problem to update it however.

I believe we send a blank accept-header because that tends to get most API's to _**not**_ gzip their responses to us which causes problems. I suspect with `format=json` the API response will come through our NGINX as expected

> The characters you're seeing in your browser aren't an issue however. This is because Chrome isn't defaulting to UTF-8 for the response. See this discussion: tomchristie/django-rest-framework#2891 Perl should handle this ok, and seems to in the test environment.

Thanks for the explanation! I realized this a little later as well testing in another browser.